pipettes/firmware/main.cpp

```diff
@@ -131,12 +132,9 @@ auto main() -> int {
     adc_init();
     auto id = pipette_mounts::detect_id();
 
-    // TODO(lc 01/04/2021): We should move this setup to another object
-    // responsible for sensor control.
-    auto i2chandler_struct = I2CHandlerStruct{};
+    I2CHandlerStruct i2chandler_struct{};
```

Comment:
This is gonna have exactly the same problem - `i2chandler_struct` also must be on the free store rather than main's stack, unless it is copied to somewhere else.